src/ofono/provider_ofono.cpp

```diff
@@ -1020,9 +1069,21 @@ class Provider : public statefs::AProvider
         : AProvider("ofono", server)
         , bus_(QDBusConnection::systemBus())
     {
+        if (!server || server->mode == statefs_provider_mode_run)
+            manager_.reset(new ModemManager(bus_));
+
+        // We need the modem order to be static, so we look for modems with _0 and _1, or phonesim.
+        // Perhaps needs a config file, or perhaps we'll get a new ofono API. This will do for now.
         auto ns = std::make_shared<MainNs>
-            (bus_, server ? server->mode : statefs_provider_mode_run);
+            (bus_, "Cellular", manager_.get(), QStringLiteral(".*(_0|phonesim)$"), server ? server->mode : statefs_provider_mode_run);
         insert(std::static_pointer_cast<statefs::ANode>(ns));
+
+        ns = std::make_shared<MainNs>
+            (bus_, "Cellular_1", manager_.get(), QStringLiteral(".*(_1|phonesim)$"), server ? server->mode : statefs_provider_mode_run);
```

Comment:
iiuc, "phonesim" should match 1st modem, should it be taken for the 2nd modem too?